src/scrabbag.js

```diff
@@ -0,0 +1,50 @@
+//input a string
+//split string into an array
+//iterate over the array
+//for each letter and subtract one
+//MAKE THE STRING UPPERCASE
+
+var scrabbleBag = {
+  A: 9,
+  B: 2,
+  C: 2,
+  D: 4,
+  E: 12,
+  F: 2,
+  G: 3,
+  H: 2,
+  I: 9,
+  J: 1,
+  K: 1,
+  L: 4,
+  M: 2,
+  N: 6,
+  O: 8,
+  P: 2,
+  Q: 1,
+  R: 6,
+  S: 4,
+  T: 6,
+  U: 4,
+  V: 2,
+  W: 2,
+  X: 1,
+  Y: 2,
+  Z: 1,
+  _: 2
+}
+
+function scrabbleRemainder(str){
+var upperCase = str.toUpperCase()
+var toArray = upperCase.split('')
+for(var i = 0; i < toArray.length; i++){
+  var currentLetter = toArray[i]
+  var remainderTiles = scrabbleBag[currentLetter]
+    remainderTiles = remainderTiles - 1
```

Comment:
This indentation is confusing - it is considered best practice, and aids in readability and maintainability, to consistently left align code in the same block, indenting only to being a new block.  Consider using whitespace between the setup (declaration of variables) and body (processing using those variables) to increase readability.

There are also some more concise array methods that could be used to accomplish this operation.  Consider:
```js
array.forEach( (letter, index) => {
  scrabbleBag[ letter ]--
})

return scrabbleBag
``` 
Also, if this is how `scrabbleBag` is being used, it should not be a global variable - any other function in this module could inadvertently modify it, leading to subtle logic errors.